editor/js/libs/app/index.html

```diff
@@ -16,13 +16,16 @@
 		</style>
 	</head>
 	<body ontouchstart="">
+		<script src="./HelioWebXRPolyfill.js"></script>
```

Comment:
Isn't `HelioWebXRPolyfill.js` placed in the `js` directory? If so, `<script src="./js/HelioWebXRPolyfill.js"></script>` would be the correct markup.

Reply:
I realized this a few seconds after committing... fixed again 😄